www/layouts/home.html

```diff
@@ -72,6 +72,7 @@ <h1 class="unlocking font-weight-bold text-white">
   {{ $coursePages := where $pages "Type" "==" "course" }}
   {{ $childPages := where $coursePages ".Params.layout" "==" "course_home" }}
   {{ partial "home_course_cards.html" $childPages }}
+  {{ partial "course_collection_tab.html" . }}
```

Comment:
Why "course collection tab" ? The design doesn't make me think of a tab. 

I think it would also be good to start the partial's name with home, similar to `home_course_cards.html`

Reply:
Resolved